apps/omg_status/lib/omg_status/metric/telemetry.ex

```diff
@@ -35,11 +35,12 @@ defmodule OMG.Status.Metric.Telemetry do
         TelemetryMetricsStatsd,
         metrics: metrics(),
         global_tags: [
+          service: service,
           version: version
         ],
         host: dd_host,
         port: dd_port,
-        prefix: "omg.#{service}",
+        prefix: "omg",
```

Comment:
I may change this to "elixir" rather than "omg". I think the important part is that these metrics come from an "elixir" app.